From 736e8d289efb5a1e749dc9961fcc4fd519fa8b4c Mon Sep 17 00:00:00 2001
From: "Richard W.M. Jones" <rjones@redhat.com>
Date: Tue, 5 Nov 2013 07:15:19 +0000
Subject: [PATCH] builder: Fix missing files in EXTRA_DIST.

This fixes commit a4800e2d4fc50f372a8b626333c4fbb9b8ca9633.

(cherry picked from commit 2035a75d66676c5fb2cc047a36c174dd953cd1b9)
---
 builder/Makefile.am | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/builder/Makefile.am b/builder/Makefile.am
index cb490e1..9dbe51e 100644
--- a/builder/Makefile.am
+++ b/builder/Makefile.am
@@ -26,6 +26,7 @@ AM_CFLAGS = \
 EXTRA_DIST = \
 	$(SOURCES) \
 	virt-builder.pod \
+	virt-index-validate.pod \
 	test-virt-builder.sh \
 	test-virt-builder-list.sh \
 	website/.gitignore \
@@ -70,6 +71,7 @@ SOURCES = \
 	get_kernel.ml \
 	index_parser.mli \
 	index_parser.ml \
+	index-parser-c.c \
 	list_entries.mli \
 	list_entries.ml \
 	perl_edit.ml \
-- 
1.8.3.1

